src/nvim/if_cscope.c

```diff
@@ -577,9 +578,8 @@ static int cs_cnt_connections(void)
   return cnt;
 } /* cs_cnt_connections */
 
-static void 
-cs_reading_emsg (
-    int idx        /* connection index */
+static void cs_reading_emsg(
+    size_t idx        /* connection index */
 )
 {
   EMSGN(_("E262: error reading cscope connection %" PRId64), idx);
```

Comment:
Anyways, if you can fix this issue, we will merge this one, and hopefully #2227 can be rebased on it.